Fix the error where gconf cannot connect to the DBus session, which causes
guest-account to terminate unsuccessfully, which causes the temporary accounts
to remain on the system and causes LightDM to restart without any error message,
which sucks :)
--- a/debian/guest-account
+++ b/debian/guest-account
@@ -71,6 +71,7 @@
   else
     cp -rT /etc/skel/ "$HOME"
   fi
+  mkdir -p "$HOME/.gconf/"
   chown -R $USER:$USER "$HOME"
   usermod -d "$HOME" "$USER"
 
@@ -81,7 +82,7 @@
   # disable screensaver, to avoid locking guest out of itself (no password)
   if [ -e /usr/bin/gconftool-2 ]; then
    su $USER <<EOF
-   gconftool-2 --set --type bool /desktop/gnome/lockdown/disable_lock_screen True
+   dbus-launch gconftool-2 --set --type bool /desktop/gnome/lockdown/disable_lock_screen True
 EOF
   fi
 
